Plans/visionOS-Swift_Base.xctestplan

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
`m`: Same as [here](https://github.com/getsentry/sentry-cocoa/pull/5167#discussion_r2073082844)

Reply:
There are no tests for visionOS-Swift.